packages/grafana-ui/src/components/Combobox/MultiCombobox.tsx

```diff
@@ -284,6 +296,23 @@ export const MultiCombobox = <T extends string | number>(props: MultiComboboxPro
           />
 
           <div className={multiStyles.suffix} ref={suffixMeasureRef} {...getToggleButtonProps()}>
+            {isClearable && selectedItems.length > 0 && (
+              <Icon
+                name="times"
+                className={styles.clear}
+                title={t('multicombobox.clear.title', 'Clear all')}
+                tabIndex={0}
+                role="button"
+                onClick={() => {
+                  reset();
+                }}
```

Comment:
you need `ev.stopPropagation()` here to prevent the menu from opening when clicking it

```ts
onClick={(ev) => {
  ev.stopPropagation();
  reset();
}}
```